controls/chat/styling.md

```diff
@@ -25,6 +25,11 @@ To customize the different parts of the control, you can use the following built
 
 - `OutgoingLabelStyle` (target type `Label`)&mdash;Defines the style of the label of the outgoing message.
 
+In addition, the following properties set the background of the Chat:
```

Comment:
You can add the background properties to the list above. You don't need to create a new list.
```suggestion
```